source/configuration_manual/mail_location/obox/amazon_s3.rst

```diff
@@ -72,6 +72,13 @@ For more information about IAM roles for EC2 please refer to:
 For general information about IAM:
 `IAM UserGuide <https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html>`_
 
+Known Issues
+^^^^^^^^^^^^
+
+.. versionchanged:: v2.3.11 In earlier versions of Dovecot ``403 Forbidden``
+   errors can occur around key rotation, approximately every 6 hours.
+   If the session that has these errors is terminated and reestablished
+   the authentication works as expected again.
```

Comment:
I think this happens only for 1 second? So not even needed to restart authentication - just wait for 1sec?

Also, do we want to add this to documentation? We generally haven't been listing bugs in documentation..

Reply:
> I think this happens only for 1 second? So not even needed to restart authentication - just wait for 1sec?

Yes, practically it happened for a very short time only but that was only with imaptest. I don't 
know how other clients react and i believe only a new session does resolve the situation. 

> Also, do we want to add this to documentation? We generally haven't been listing bugs in documentation..

I thought it would be good to prevent people setting up everything correctly and then being surprised by after 6 hours.

I can close it anyways if you think so :)

Reply:
But the cache check was using ioloop_time. So even with long-running sessions the ioloop_time should have changed, invalidated the cache and fixed this to work again. Right?..

Reply:
Yes it uses ioloop_time but only the date. So after a day change it works again if a session is living that long, but changes of ioloop_time within a day don't have any influence.

Reply:
Oh, right. Somehow I glossed over the timestamp format part.